File: java/FizzBuzz.java
public class FizzBuzz {
	public static void main(String[] args) { 
		new FizzBuzz(3,5).run(1,100);
	}
	
	private int fizzdiv;
	private int buzzdiv;

	public FizzBuzz(int fd, int bd) {
		fizzdiv = fd; 
		buzzdiv = bd;
	}

	private void run(int start, int end) {
		for (int i=start; i <= end; ++i) {
			if (i % fizzdiv == 0 && i % buzzdiv == 0)
				System.out.println("FizzBuzz");
			else if (i % fizzdiv == 0)
				System.out.println("Fizz");
			else if (i % buzzdiv == 0)
				System.out.println("Buzz");
			else 
				System.out.println(i);
		}
	}
}

File: python/fizzbuzz.py
#! /usr/bin/env python

fizzdiv = 3
buzzdiv = 5

def fizzbuzzify(num):
   if (num % fizzdiv == 0) and (num % buzzdiv == 0):
      return "fizzbuzz"
   elif num % fizzdiv == 0:
      return "fizz"
   elif num % buzzdiv == 0:
      return "buzz"
   else:
      return str(num)

def makefblist(start,finish):
   return [fizzbuzzify(x) for x in range(start,finish+1)]

def printfblist(start,finish):
   for x in makefblist(start,finish):
      print x

def fizzbuzz():
   """Standard 1 to 100 fizz buzz printer"""
   printfblist(1,100)


fizzbuzz()
